ui/src/styles/styles.scss

```diff
@@ -88,6 +88,9 @@ label {
   .jst-md-center {
     justify-content: center;
   }
+  .width-100 {
```

Comment:
should we make it more specific for this to take effect only on `DraftPreviewer`?